CraftingRevisions/CraftingRevisions.csproj

```diff
@@ -1,84 +1,35 @@
-      <Private>False</Private>
-    </Reference>
-    <Reference Include="System" />
-    <Reference Include="UnhollowerBaseLib, Version=0.4.16.2, Culture=neutral, processorArchitecture=MSIL">
-      <SpecificVersion>False</SpecificVersion>
-      <Private>False</Private>
-    </Reference>
-    <Reference Include="UnityEngine.CoreModule, Version=0.0.0.0, Culture=neutral, processorArchitecture=MSIL">
-      <Private>False</Private>
-    </Reference>
-  </ItemGroup>
-  <ItemGroup>
-    <Compile Include="BlueprintMapper.cs" />
-    <Compile Include="BlueprintManager.cs" />
-    <Compile Include="BuildInfo.cs" />
-    <Compile Include="Exceptions\InvalidBlueprintException.cs" />
-    <Compile Include="ModCraftingLocation.cs" />
-    <Compile Include="CraftingMenu\CraftingExtensions.cs" />
-    <Compile Include="CraftingMenu\CraftingMenuPatches.cs" />
-    <Compile Include="CraftingMenu\MethodReplacements.cs" />
-    <Compile Include="CraftingMenu\ModCraftingCategory.cs" />
-    <Compile Include="CraftingMenu\RecipeDisplayItem.cs" />
-    <Compile Include="CraftingMenu\RecipeMenuGUI.cs" />
-    <Compile Include="CraftingRevisionsMod.cs" />
-    <Compile Include="ModBlueprintData.cs" />
-    <Compile Include="Patches\GameManager_Awake.cs" />
-    <Compile Include="Patches\OverrideCraftingResultConditionPatches.cs" />
-    <Compile Include="Properties\AssemblyInfo.cs" />
-    <Compile Include="Recipes\ModIngredient.cs" />
-    <Compile Include="Recipes\ModPowderIngredient.cs" />
-    <Compile Include="Recipes\ModPowderIngredientType.cs" />
-    <Compile Include="Recipes\ModRecipe.cs" />
-    <Compile Include="Settings.cs" />
-    <Compile Include="ModSkillType.cs" />
-  </ItemGroup>
-  <Import Project="$(MSBuildToolsPath)\Microsoft.CSharp.targets" />
-</Project>
\ No newline at end of file
+﻿<Project Sdk="Microsoft.NET.Sdk">
+	<!--This is an xml comment. Comments have no impact on compiling.-->
+
+	<PropertyGroup>
+		<!--This is the .NET version the mod will be compiled with. Don't change it.-->
+		<TargetFramework>net6.0</TargetFramework>
+
+		<!--This tells the compiler to use the latest C# version.-->
+		<LangVersion>Latest</LangVersion>
+
+		<!--This adds global usings for a few common System namespaces.-->
+		<ImplicitUsings>enable</ImplicitUsings>
+
+		<!--This enables nullable annotation and analysis. It's good coding form.-->
+		<Nullable>enable</Nullable>
+
+		<!--This tells the compiler to use assembly attributes instead of generating its own.-->
+		<GenerateAssemblyInfo>false</GenerateAssemblyInfo>
+
+		<!--PDB files give line numbers in stack traces (errors). This is useful for debugging. There are 3 options:-->
+		<!--full has a pdb file created beside the dll.-->
+		<!--embedded has the pdb data embedded within the dll. This is useful because bug reports will then have line numbers.-->
+		<!--none skips creation of pdb data.-->
+		<DebugType>embedded</DebugType>
+	</PropertyGroup>
+
```

Comment:
Can you add Nuget information? It should be similar to https://github.com/zeobviouslyfakeacc/ModSettings/blob/master/ModSettings.csproj#L25-L42